osquery/tables/system/windows/authenticode.cpp

```diff
@@ -125,6 +126,49 @@ void generateRow(Row& row, const SignatureInformation& signature_info) {
   }
 }
 
+__checkReturn __success(return == true) bool getCatalogPathForFilePath(
```

Comment:
Do we need the `__checkReturn`, `__in`, etc annotations for this to work? This is not a convention we are using in the code base and I'd like to avoid setting precedent here without a plan to discuss or adopt it throughout.

Reply:
My bad-- I'll take out. I forgot to remove prior to submitting. I use SAL for code analysis.

Reply:
Ah, gotya